infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationSession.java

```diff
@@ -0,0 +1,69 @@
+package org.corfudb.infrastructure.logreplication.infrastructure;
+
+import lombok.Getter;
+import org.corfudb.infrastructure.logreplication.LogReplicationConfig;
+
+import java.util.Objects;
+
+import static org.corfudb.infrastructure.logreplication.infrastructure.plugins.DefaultLogReplicationConfigAdapter.SAMPLE_CLIENT;
+
+/**
+ * This class represents a session/connection with a remote cluster.  Each session has a dedicated Replication FSM and
+ * Snapshot and LogEntry Sync Readers/Writers.
+ *
+ * A session is uniquely identified with the following parameters:
+ * 1) Cluster Id of the Remote Cluster
+ * 2) Replication Model being used by the Remote Cluster
```

Comment:
I do not see the Replication Model in the actual class ?

Reply:
It is in the Subscriber class which is a member of this class.  Updated the comment accordingly.